frontend/src/components/utils/autoRefreshHandler.tsx

```diff
@@ -33,15 +33,7 @@ export default (
     };
 
     componentDidMount() {
-      this.timer = setTimeout(this.regularFetchInfo, 0);
-    }
-
-    componentWillUnmount() {
-      const timer = this.timer;
-      this.timer = null;
-      if (timer !== null) {
-        clearTimeout(timer);
-      }
```

Comment:
Won't this break pages like Account Details, Block Details?

Reply:
We never apply autoRefreshHandler to details pages, we only use it on list pages